unity-client/Assets/Scripts/MainScripts/DCL/Helpers/NFTHelper/NFTMarkets/OpenSea_Internal/Request.cs

```diff
@@ -269,40 +278,46 @@ IEnumerator Fetch()
-                        Debug.Log($"RequestGroup: Request to OpenSea {url}");
-                    yield return request.SendWebRequest();
+                if (OpenSeaRequestController.VERBOSE)
+                    Debug.Log($"RequestGroup: Request to OpenSea {url}");
 
-                    AssetsResponse response = null;
+                // NOTE(Santi): In this case, as this code is implementing a very specific retries system (including delays), we use our
+                //              custom WebRequest system without retries (requestAttemps = 1) and let the current code to apply the retries.
+                WebRequestAsyncOperation requestOp = WebRequestController.i.Get(url: url, requestAttemps: 1, disposeOnCompleted: false);
+                yield return requestOp;
 
-                    if (!request.isNetworkError && !request.isHttpError)
-                    {
-                        response = Utils.FromJsonWithNulls<AssetsResponse>(request.downloadHandler.text);
-                    }
+                AssetsResponse response = null;
+                if (requestOp.isSucceded)
+                {
+                    response = Utils.FromJsonWithNulls<AssetsResponse>(requestOp.webRequest.downloadHandler.text);
+                }
 
-                    if (OpenSeaRequestController.VERBOSE)
-                        Debug.Log($"RequestGroup: Request resolving {response != null} {request.error} {url}");
-                    if (response != null)
+                if (OpenSeaRequestController.VERBOSE)
+                    Debug.Log($"RequestGroup: Request resolving {response != null} {requestOp.webRequest.error} {url}");
+
+                if (response != null)
+                {
+                    shouldRetry = false;
+                    OnSuccess?.Invoke(response);
+                    requestOp.Dispose();
+                }
+                else
+                {
+                    shouldRetry = retryCount < REQUEST_RETRIES;
+                    if (!shouldRetry)
                     {
-                        shouldRetry = false;
-                        OnSuccess?.Invoke(response);
+                        OnError?.Invoke("Error " + requestOp.webRequest.downloadHandler.text);
+                        requestOp.Dispose();
                     }
                     else
                     {
-                        shouldRetry = retryCount < REQUEST_RETRIES;
-                        if (!shouldRetry)
-                        {
-                            OnError?.Invoke("Error " + request.downloadHandler.text);
-                        }
-                        else
-                        {
-                            retryCount++;
-                            if (OpenSeaRequestController.VERBOSE)
-                                Debug.Log($"RequestGroup: Request retrying {url}");
-                            yield return new WaitForSeconds(GetRetryDelay());
-                        }
+                        retryCount++;
```

Comment:
We should remove the retryCount to favor our retry logic here.

Reply:
I left it that way on purpose. After a conversation with @pbosio about this specific case, we decided to keep the specific "retries" implementation there. Please take a loook to my comment here: https://github.com/decentraland/explorer/blob/f8bbae8c344d097818cbcded8b50dc3a201a8ed2/unity-client/Assets/Scripts/MainScripts/DCL/Helpers/NFTHelper/NFTMarkets/OpenSea_Internal/Request.cs#L147

Reply:
The reason is that requests for OpenSea handle a very specific retry behavior added by @AjimenezDCL We shouldn't change that. (after a request has failed the request is splitted in half and 2 request are send and so on and on...)

Reply:
Ok. Resolved.